src/app/(Authentication)/SignUp/Email/index.tsx

```diff
@@ -42,10 +43,17 @@ export default function SignUpScreen() {
     }
   };
 
-  const handleSubmit = () => {
+  const handleSubmit = async () => {
     if (validateName() && validateEmail()) {
+      const { error } = await supabase.auth.signInWithOtp({
```

Comment:
On the [supabase docs](https://supabase.com/docs/reference/javascript/auth-signinwithotp) I found this:

 "If the user doesn't exist, signInWithOtp() will signup the user instead. To restrict this behaviour, you can set shouldCreateUser in SignInWithPasswordlessCredentials.options to false."
 
 I imagine they have an option for verifying a OTP without officially registering a user, but I'd have to dig deeper into the docs.

Reply:
added but haven't tested due to rate limiting, will work on tmrw